src/view/plandetail/CollageTemplate.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Divider, HStack, Image, Text, VStack } from "@chakra-ui/react";
+
+type CollagePlace = {
+    name: string;
+    duration: number;
+    imageUrl: string;
+};
+
+type CollageProps = {
+    title: string;
+    places: CollagePlace[];
+    introduction: string;
+};
+
+export function CollageTemplate({ title, places, introduction }: CollageProps) {
+    return (
+        <Box padding={16}>
+            <Box textAlign="center" mb={4}>
+                <Text
+                    color="rgba(130, 141, 205, 1)"
+                    fontFamily="Inter"
+                    fontWeight="bold"
+                    fontSize="80px"
+                >
+                    {title}
+                </Text>
+            </Box>
+            <Divider borderColor="#A4ABD4" my={4} />
+            {places.map((place, index) => (
+                <Box key={index}>
+                    <HStack
+                        justifyContent="space-between"
+                        alignItems="flex-start"
+                        flexDirection={index % 2 === 0 ? "row" : "row-reverse"}
+                    >
+                        <Image
+                            src={place.imageUrl}
+                            alt={`Collage Image ${index}`}
+                            borderRadius="lg"
+                            boxSize="400px"
+                        />
+                        <VStack alignItems="flex-start">
```

Comment:
[imo]「ぴちっ」とくっついていたほうが、まとまって見えると思います！
```suggestion
                        <VStack alignItems="flex-start" spacing={0}>
```
![image](https://github.com/poroto-app/poroto/assets/55840281/785b901b-5893-4129-ae1b-2b1cdcdb1c30)